js/search.js

```diff
@@ -38,15 +38,28 @@
             table.empty();
             for (let i = 0; i < data["books"].length; i++) {
                 var icon = "<img  class='icon' src='" + data["books"][i]["image"] + "'/>";
-                var link = "<a href='" + data["books"][i]["link"] + "'>Download link</a>";
+                var link = "<a href='" + data["books"][i]["link"] + "'>Buy link</a>";
+                var id = data["books"][i]["id"];
+                var editButton = "<button " +
+                    "onclick=\"location.href ='/admin/book/edit/" + id + "'\" class='btn btn-info btn-md'>" +
+                    " <span class='glyphicon glyphicon-edit'></span>Edit</button>";
+
+                var deleteButton = "<button " +
+                    "onclick=\"location.href ='/admin/book/delete/" + id + "'\" class='btn btn-danger btn-md'>" +
+                    " <span class='glyphicon glyphicon-remove'></span>Delete</button>";
                 table.append($('<tr>')
                     .append('<td>' + icon + '</td>')
                     .append('<td>' + data["books"][i]["title"] + '</td>')
                     .append('<td>' + link + '</td>')
                     .append('<td>' + data["books"][i]["authors"] + '</td>')
-                    .append('<td></td>'));
+                    .append('<td>' + data["books"][i]["price"] + '</td>')
+                    .append('<td>' + data["books"][i]["currency"] + '</td>')
+                    .append('<td>' + data["books"][i]["language"] + '</td>')
+                    .append('<td><pre>' + editButton + " | " + deleteButton + '</pre></td>'));
```

Comment:
?! why ```<pre>``` ?! you simply need to put two links there, not buttons, with a | between them.